mollie-payments-for-woocommerce/includes/mollie/wc/gateway/abstract.php

```diff
@@ -445,8 +447,10 @@ public function process_payment( $order_id ) {
 			$data = apply_filters( 'woocommerce_' . $this->id . '_args', $data, $order );
 
 			try {
-				Mollie_WC_Plugin::debug( $this->id . ': Subscription switch started, fetching mandate(s) for order #' . $order_id );
-				$mandates = Mollie_WC_Plugin::getApiHelper()->getApiClient( $test_mode )->customers->get( $customer_id )->mandates();
+                Mollie_WC_Plugin::debug(
+                    "{$this->id}: Subscription switch started, fetching mandate(s) for order #{$orderId}"
+                );
+                $mandates = Mollie_WC_Plugin::getApiHelper()->getApiClient( $test_mode )->customers->get( $customer_id )->mandates();
```

Comment:
I saw you removing extra spaces around function arguments. Why are they kept here?

Reply:
@XedinUnknown As said in other comments.